core/templates/dev/head/pages/exploration_player/ImagePreloaderService.js

```diff
@@ -38,9 +40,24 @@ oppia.factory('ImagePreloaderService', [
     var _init = function(exploration) {
       _exploration = exploration;
       _states = exploration.states;
+      _getDimensionsOfImages();
       _hasImagePreloaderServiceStarted = true;
     };
 
+    /**
+     * Gets the dimensions of the images in the exploration.
+     */
+    var _getDimensionsOfImages = function() {
+      _imageDimensions = {};
+      var states = (_states.getStateNames()).map(function(stateName){
+        return _states.getState(stateName);
+      });
+      states.forEach(function(state){
+        _imageDimensions = Object.assign(_imageDimensions,
```

Comment:
ditto (we don't need `_imageDimensions = `)

Reply:
Done!